cli/src/commands/runTests.js

```diff
@@ -43,7 +43,7 @@ async function getTestGroups(repoDirPath): Promise<Array<TestGroup>> {
   const libDefs = await getLibDefs(repoDirPath);
   return libDefs.map(libDef => {
     const groupID =
-      `${libDef.pkgName}_${libDef.pkgVersionStr}--flow_` +
+      `${libDef.pkgName}_${libDef.pkgVersionStr}${path.sep}flow_` +
```

Comment:
@jeffmo See line 194 below. The reason for the change is that `libDef.pkgName` can now also contain `/`, so I'm just escaping in one place instead of two.